website/api/zkvm/precompiles.md

```diff
@@ -120,7 +120,7 @@ a message within the zkvm where an observer can measure the proving time or view
 [commit-lockfile]: https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html
 [discord-url]: https://discord.gg/risczero
 [ecdsa]: https://github.com/risc0/risc0/tree/release-1.2/examples/ecdsa
-[ecdsa-patched]: https://github.com/risc0/risc0/blob/release-1.2/examples/ecdsa/methods/guest/Cargo.toml#L13-L18
+[ecdsa-patched]: https://github.com/risc0/risc0/blob/release-1.2/examples/ecdsa/k256/Cargo.toml
```

Comment:
```suggestion
[ecdsa-patched]: https://github.com/risc0/risc0/blob/release-1.2/examples/ecdsa/k256/methods/guest/Cargo.toml#L13-L18
```

Reply:
@ylmin if you want this to land, please implement this fix -- otherwise i'll open a separate PR